ambari-web/test/views/main/service/info/metrics_view_test.js

```diff
@@ -214,19 +217,18 @@ describe('App.MainServiceInfoMetricsView', function() {
       mock.sortable.restore();
     });
 
-    it("on() should be called", function() {
+    it("MutationObserver callback should be called", function() {
       view.makeSortable('#widget_layout');
-      expect(mock.on.calledWith('DOMNodeInserted', '#widget_layout')).to.be.true;
-    });
+      const observer = new MutationObserver(mock.callback);
```

Comment:
Although PhantomJS supports `MutationObserver`, the internal implementation is inconsistent, leading to the loss of some APIs and causing test failures.

Reply:
Thank you for the suggestion, hopefully this should work consistently on phantomjs-2+